test/setup

```diff
@@ -70,8 +70,9 @@ popd
 rm -rf ${CERT_DIR}
 
 echo "booting ${VMS} by heat, then copy files for pre-deployment"
-extra_args="--ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name}"
-ursula ${extra_args} "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@"
+sleep 30
+extra_args="--ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name} --heat-stack-ssh-username=${LOGIN_USER} --heat-parameters image=${IMAGE_ID}"
+ursula ${extra_args} "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@" -vvvv
 
```

Comment:
The login name for centos images is obviously not "ubuntu" so I had to add an argument to `ursula-cli` to let me pass in the login user name I want to use (this is simple for clouds where vms are all booted with the same image, does not work for clouds that use multiple images)